src/Microsoft.AspNetCore.Mvc.Core/Properties/Resources.Designer.cs

```diff
@@ -809,6 +809,22 @@ internal static string FormatModelBinding_MissingBindRequiredMember(object p0)
             => string.Format(CultureInfo.CurrentCulture, GetString("ModelBinding_MissingBindRequiredMember"), p0);
 
         /// <summary>
+        /// A non-empty request body is required.
+        /// </summary>
+        internal static string ModelBinding_MissingRequestBodyRequiredMember
+        {
+            get { return GetString("ModelBinding_MissingRequestBodyRequiredMember"); }
+        }
+
+        /// <summary>
+        /// A non-empty request body is required.
+        /// </summary>
+        internal static string FormatModelBinding_MissingRequestBodyRequiredMember()
+        {
+            return GetString("ModelBinding_MissingRequestBodyRequiredMember");
+        }
```

Comment:
Ah, thanks for letting me know - didn't know that facility existed! Done.

(For anyone else referring to this, the syntax is `build.cmd /t:resx`.)